AppSearchSample/app/src/main/java/com/android/example/appsearchsample/NoteListItemAdapter.kt

```diff
@@ -54,18 +51,26 @@ class NoteListItemAdapter(private val onDelete: (SearchResult?) -> Unit) :
 
   /** ViewHolder for [NoteListItemAdapter]. */
   inner class NoteViewHolder(
-    binding: ItemNoteBinding,
-  ) : RecyclerView.ViewHolder(binding.root) {
-    private val noteTextView = binding.noteText
-    private val noteDeleteButtonView = binding.noteDeleteButton
+    view: View,
+  ) : RecyclerView.ViewHolder(view) {
+    val noteTextView: TextView
+    val noteDeleteButtonView: Button
+
+    init {
+      noteTextView = view.findViewById(R.id.note_text)
```

Comment:
Are we able to combine the declaration and assignment for these to make it even simpler? i.e.
```
val noteTextView: TextView = view.findViewById(R.id.note_text)
```

Reply:
Done.